src/react-vite/01-intro-to-react/intro-to-react.md

```diff
@@ -8,45 +8,41 @@
 
 ## Overview
 
-TODO
-
-## Objective 1
-
-TODO
-
-### Key concepts
-
-TODO
-
-#### Concept 1
-
-TODO
-
-#### Concept 2
-
-TODO
-
-### Requirements
-
-TODO
-
-### Setup
-
-TODO
-
-### Verify
-
-TODO
-
-### Solution
-
-<details>
-<summary>Click to see the solution</summary>
-
-TODO
-
-</details>
-
-## Next steps
-
-TODO
\ No newline at end of file
+React first appeared in 2013 with "components" as the basic building block for
+constructing a user interface. Initially components were created using
+JavaScript classes which implemented an interface that defined lifecycle
+methods. One of React's notable characteristics has existed from the very
+beginning: the use of [JSX
+syntax](https://react.dev/learn#writing-markup-with-jsx) to define the structure
```

Comment:
I think we should link to our own Academy pages for each of the key terms in this explainer, e.g. components, JSX, etc.